src/index.tsx

```diff
@@ -117,28 +126,53 @@ class SQLiteLoggerImpl {
  
     // Override console methods
     console.debug = (...args: any[]) => {
-      this.debug(util.format(...args));
-      this._originalConsole?.debug(...args);
+      if (isLogOptions(args[0])) {
+        this.debug(util.format(...args.slice(1)), args[0].tag);
+        this._originalConsole?.debug(...args.slice(1));
+      } else {
+        this.debug(util.format(...args));
+        this._originalConsole?.debug(...args);
+      }
```

Comment:
Each `console.<level>` override contains nearly identical logic. Not required, but this could be refactored to reduce duplication.